app/(landing)/components/hero.tsx

```
@@ -52,6 +61,61 @@ export const Hero = () => {
+            </p>
+            <div className="mt-2 flex justify-end">
+              <Button variant="outline" size="sm" className="cursor-pointer shadow-none" asChild>
+                <Link href={routes.blocks.base}>
+                  <span>Explore Blocks</span>
+                  <ArrowRightIcon className="!size-3" />
+                </Link>
+              </Button>
+            </div>
+          </TiltCard>
+          <div className="-end-18 absolute top-8">
+            <svg className="-rotate-30 h-16 fill-foreground/50" viewBox="0 0 511 464">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)